pkg/rancher-desktop/utils/shortcuts.ts

```diff
@@ -0,0 +1,207 @@
+import os from 'os';
+
+import { BrowserWindow } from 'electron';
+
+import { toArray } from '@pkg/utils/array';
+
+type Platform = 'darwin' | 'linux' | 'win32';
+
+const Shortcut = {
+  /**
+   * TODO implement shift, alt
+   */
+  key: (shortcut: Shortcut | Electron.Input) => {
+    return `${ shortcut.meta ? 'Cmd+' : '' }${ shortcut.control ? 'Ctrl+' : '' }${ shortcut.key }`;
+  },
+};
+
+export interface Shortcut {
+  platform?: Platform | Platform[];
+  meta?: boolean;
+  shift?: boolean;
+  control?: boolean;
+  alt?: boolean;
+  key: string;
+}
+
+interface ShortcutExt extends Shortcut {
+  callback: () => void;
+}
+
+function matchPlatform(shortcut: Shortcut): boolean {
+  if (shortcut.platform) {
+    return toArray(shortcut.platform).includes(os.platform());
+  }
+
+  return true;
+}
+
+function currentPlatform(shortcuts: Shortcut[], action: (s: Shortcut) => void) {
+  shortcuts.forEach((s) => {
+    if (matchPlatform(s)) {
+      action(s);
+    }
+  });
+}
+
+function log(level: 'info' | 'warn' | 'error', msg: string) {
+  console[level](`shortcuts - ${ msg }`);
+}
```

Comment:
Should we be using the logging util (`pkg/rancher-desktop/utils/logging.ts`) instead?

Reply:
Yes, better using the logging util.